Lib/test/test_interpreters.py

```diff
@@ -781,6 +781,8 @@ def test_default_shareables(self):
                 True,
                 False,
                 100.0,
+                (),
+                (1, True),
```

Comment:
Yeah, putting the empty tuple + non-empty case here makes sense.  Consider including nesting in the non-empty case, to demonstrate the extent of complexity:

```suggestion
                (),
                (1, ('spam', 'eggs'), True),
```